### ISSUE TITLE
Research next.js vs express.js and decide which to use

### ISSUE THREAD

**Issue by jxz12** [This-team-CFY/classplanner#8]
next.js makes deployments easier, but may be difficult to learn a new framework.

**Comment by AnnaFYZ** [comment]
Cons
- **Requires an in-house development team to handle development and management.**
- Avails a few plugins
- Developers need to create the complete front-end layer since Next.js does not include the installed front end pages. This elevates the cost of flexibility.
- **Does not have a built-in state manager. You need to use state containers like Redux or MobX.**
- Longer development time since it supports static web development

**Comment by jxz12** [comment]
I found a nice tutorial here, see what you think!
https://www.freecodecamp.org/news/build-a-full-stack-application-with-nextjs/

**Comment by AnnaFYZ** [comment]
I think this part "To get started with this tutorial, I highly recommend
using the provided starter boilerplate
<https://github.com/Yazdun/next-fcc-familyguy/tree/starter> that I
specifically created for this tutorial. It saves your valuable time by
already including the necessary dependencies and folder structure,
eliminating the need to set up your project from scratch."
tells us already where we're going to stack.
Setting up from scratch a thing we don't have an experience with will take
a huge amount of time

пн, 13 лист. 2023 р., 22:27 користувач Jonathan Zheng <
***@***.***> пише:

**Comment by jxz12** [comment]
I agree in general that we should just stick with express.js for now. Another benefit of express.js is that you get practice calling endpoints from the UI which is probably good experience anyway.
